### PR TITLE
Chore: set schema_version default value directly in the pydantic model

### PR DESCRIPTION
**Pull request by GeorgeSittas**

### CODE REVIEW

File: sqlmesh/core/state_sync/engine_adapter.py [hunk 1]
```diff
@@ -485,9 +485,7 @@ def _get_versions(self, lock_for_update: bool = False) -> Versions:
             return no_version
 
         return Versions(
-            schema_version=row[0],
-            sqlglot_version=row[1],
-            sqlmesh_version=seq_get(row, 2) or "0.0.0",
+            schema_version=row[0], sqlglot_version=row[1], sqlmesh_version=seq_get(row, 2)
```
Comment (tobymao) [inline sqlmesh/core/state_sync/engine_adapter.py:488]: same with line 474?

Comment (GeorgeSittas) [reply sqlmesh/core/state_sync/engine_adapter.py:488]: should we then apply defaults to sqlglot_version and schema_version too?

Comment (tobymao) [reply sqlmesh/core/state_sync/engine_adapter.py:488]: sure

Comment (GeorgeSittas) [reply sqlmesh/core/state_sync/engine_adapter.py:488]: updated

**Review by tobymao — APPROVED** [review]

**Pull request merged**